php/templates/configmap-fpm.yaml

```diff
@@ -1,6 +1,24 @@
 ---
 apiVersion: v1
 kind: ConfigMap
+metadata:
+  name: {{ template "php.fullname" . }}-fpm-phpini
+  labels:
+    app: {{ template "php.name" . }}
+    chart: {{ template "php.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    checksum/fpm-php-fpm-conf: {{ tpl (index .Values.fpm.templates "php.ini" | toYaml) . | sha256sum }}
```

Comment:
I can see that `checksum/fpm-php-fpm-conf` is already used below.
https://github.com/chatwork/charts/blob/fe608c40b38166b59a0b2e68b92f5f9f400f03a6/php/templates/configmap-fpm.yaml#L30